dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/trackedentity/TrackedEntityStoreTest.java

```diff
@@ -329,8 +351,18 @@ void testStartsWithQueryOperator() {
         new TrackedEntityAttributeValue(atA, trackedEntityE, "Male"));
     attributeValueService.addTrackedEntityAttributeValue(
         new TrackedEntityAttributeValue(atA, trackedEntityF, "Female"));
-    enrollmentService.enrollTrackedEntity(trackedEntityB, prA, new Date(), new Date(), ouB);
-    enrollmentService.enrollTrackedEntity(trackedEntityE, prA, new Date(), new Date(), ouB);
+
+    Enrollment enrollmentA = createEnrollment(prA, trackedEntityB, ouB);
```

Comment:
You seem to be doing that multiple times. You can create a `enrollTrackedEntity` in the test class so that the tests remain "as readable".